### PR TITLE
Added continuous-token as response

### PR DESCRIPTION
**Pull request by roshan-sy**
Please make sure the code is following contribution guidelines in CONTRIBUTING.md

 - [x] : This PR has a corresponding issue open in the Repository.
 - [x] : Approach is signed off on the issue.

Fixes: #1112 

Modified the response to include **continuous-token**

### CODE REVIEW

File: azure-devops/azext_devops/devops_sdk/client.py [hunk 1]
```diff
@@ -102,6 +102,7 @@ def _send(self, http_method, location_id, version, route_values=None,
         if Client._session_header_key in Client._session_data and Client._session_header_key not in headers:
             headers[Client._session_header_key] = Client._session_data[Client._session_header_key]
         response = self._send_request(request=request, headers=headers, content=content, media_type=media_type)
+        response.continuation_token = response.headers.get('X-MS-ContinuationToken')
```
Comment (aasiddiq) [inline azure-devops/azext_devops/devops_sdk/client.py:105]: Maybe it is included someplace else but when we are making a subsequent call don't we need to send the continuation token in request as well?

Comment (roshan-sy) [reply azure-devops/azext_devops/devops_sdk/client.py:105]: Checking

Comment (roshan-sy) [reply azure-devops/azext_devops/devops_sdk/client.py:105]: @aasiddiq 

It is not there actually, the "invoke" command does not have support for continuous token.
We can close this one and I can create a separate "Feature Request" to support "continuous token" in the request header.
And let's track it separately.

**Comment by roshan-sy** [comment]
Canceling pool request as more changes are required

**Comment by roshan-sy** [comment]
Adding sample requests with Changes

image
image

File: azure-devops/azext_devops/devops_sdk/client.py (hunk 1 above)
Comment (roshan-sy) [reply azure-devops/azext_devops/devops_sdk/client.py:109]: "Clarification"
Invoke supports continuation token but not in the header, It support as a Query string.

File: azure-devops/azext_devops/dev/team/invoke.py [hunk 2]
```diff
@@ -114,7 +114,10 @@ def invoke(area=None, resource=None,
         logger.info('Content type header is None.')
         is_content_available = False
     elif 'json' in response.headers.get("content-type") and not out_file:
-        return response.json()
+        response_dict = response.json()
+        response_dict["continuation_token"] = response.continuation_token
```
Comment (aasiddiq) [inline azure-devops/azext_devops/dev/team/invoke.py:118]: Since we are adding the token to the response in `client.py` do we still need this?

**Review by aasiddiq — APPROVED** [review]

**Pull request merged**